### PR TITLE
Itemsets not avaiable for community plan

### PR DESCRIPTION
**Pull request by emord**
Going to be pretty tough to use itemsets in vellum if you can't create lookup tables

cc @czue

### CODE REVIEW

**Comment by czue** [comment]
legit.

File: corehq/apps/app_manager/views.py [hunk 1]
```diff
@@ -1232,7 +1232,8 @@ def form_designer(request, domain, app_id, module_id=None, form_id=None,
                             unique_form_id=form.unique_id)
 
     vellum_plugins = ["modeliteration"]
-    if toggles.VELLUM_ITEMSETS.enabled(domain):
+    if (toggles.VELLUM_ITEMSETS.enabled(domain) and request.subscription):
+        # request.subscription checks it's not a community subscription
```
Comment (czue) [inline corehq/apps/app_manager/views.py]: is this pattern used elsewhere?

Comment (emord) [inline corehq/apps/app_manager/views.py]: @nickpell mentioned this was how Community subscriptions were represented. Should I be using a different pattern?

Comment (czue) [inline corehq/apps/app_manager/views.py]: no idea, was just wondering if this was the way it's done everywhere. sounds like yes

Comment (nickpell) [inline corehq/apps/app_manager/views.py]: @emord You want to follow this pattern: https://github.com/dimagi/commcare-hq/pull/5750

Sometimes community subscriptions exist explicitly, other times the subscription is just ```None```

Comment (czue) [inline corehq/apps/app_manager/views.py]: right, that makes a lot more sense and is something i should have realized/thought harder about

**Pull request merged**